jha/src/services/api-service.js

```diff
@@ -43,6 +44,18 @@ class ApiService {
       authorizedBySpouse: null, // Set below.
       spouse: null, // Set below.
     };
+    
+    // only mail address is required for FPCare-only submissions
+    if (onlyApplyingForFPCare) {
+      jsonPayload.addressLine1 = formState.mailAddressLine1 || null;
+      jsonPayload.addressLine2 = formState.mailAddressLine2 || null;
+      jsonPayload.addressLine3 = formState.mailAddressLine3 || null;
+      jsonPayload.city = formState.mailCity || null;
+      jsonPayload.postalCode = formState.mailPostalCode || null;
```

Comment:
You need to strip the spaces from the postal code (use the `stripSpaces` helper).

Reply:
Done, rerequested your review